### ISSUE TITLE
Cannot toggle repeatedly through yankring

### ISSUE THREAD

**Issue by simlei** [svermeulen/vim-easyclip#119]
I past text with ``p``, then immediately press ``CTRL-P``. It goes back one entry in the buffer. I paste, the entry that was before the first one is pasted. So far, so good.

But then it gets sad:
Pressing ``CTRL-P`` multiple times after a paste only toggles the yankring revolution once! At the second ``CTRL-P``, I get the status: "last action was not paste, swap ignored".

But I want to access the yank 2 times before my last!
What am I doing wrong? Or is it even a bug...

In the README it was described as 
```
[...] What this means for example is that you can toggle a given paste back using <CTRL-P> 10 times, [...]
```

So, I assumed, repeated CTRL-P was the way to go.

Thanks!

**Comment by svermeulen** [comment]
Yes, this is a known issue, reported here as well : https://github.com/svermeulen/vim-easyclip/issues/109

Unfortunately I don't have time to look into it for another couple weeks minimum.  Like I said in the other issue, if anyone with vimscript knowlege is able to help in the meantime that would be great

**Comment by simlei** [comment]
Is there any news on this? Your plugin in concept is great, but right now I have fallen back on the Yankring which massively pollutes my key bindings. I absolutely do not have the skills to debug a vimscript plugin that deals with registers, though :/

**Comment by svermeulen** [comment]
No news yet, but like I said in my last comment, I should finally have time in the next week to start taking a look

I am not seeing this issue so I think it's something new to Vim 8 / Neovim?  Not sure

**Comment by simlei** [comment]
I'm sandboxing my vim right now anyways and should be able to make a minimal reproduction repo in the next week. You would just have to make it and a bashscript would bootstrap 1) subfolder-installed vim 2) minimal vimrc via VIMINIT environment variable. I'll post again here when I have it.

**Comment by razor-x** [comment]
I tired rolling back to some earlier commits when I believe this was working for me on Neovim, but no luck. I'm guessing that means that something changed in Neovim that broke this functionality.

**Comment by gauteh** [comment]
I'm hitting this bug as well. Would be super nice if this could be fixed! Missing the other features from similar plugins.

**Comment by svermeulen** [comment]
As stated now at the top of the [readme](https://github.com/svermeulen/vim-easyclip/blob/master/README.md), easyclip is being split up into three new plugins that target Vim 8 / Neovim.  I'm migrating to neovim and have got the previous yank swapping functionality working correctly again in Neovim with nvim-yoink
